2017/opmodes/DaisyLaunchAutonomous.java

```diff
@@ -52,27 +78,52 @@ public void init()
-        deadReckonTask = new DeadReckonTask(this, path);
+        conveyor = hardwareMap.dcMotor.get("conveyor");
 
         runToPositionTask = new RunToEncoderValueTask(this, launcher, LAUNCH_POSITION, 1.0);
 
+        frontLeft.setMode(DcMotor.RunMode.STOP_AND_RESET_ENCODER);
+        frontLeft.setMode(DcMotor.RunMode.RUN_USING_ENCODER);
+        frontRight.setMode(DcMotor.RunMode.STOP_AND_RESET_ENCODER);
+        frontRight.setMode(DcMotor.RunMode.RUN_USING_ENCODER);
+        rearLeft.setMode(DcMotor.RunMode.STOP_AND_RESET_ENCODER);
+        rearLeft.setMode(DcMotor.RunMode.RUN_USING_ENCODER);
+        rearRight.setMode(DcMotor.RunMode.STOP_AND_RESET_ENCODER);
+        rearRight.setMode(DcMotor.RunMode.RUN_USING_ENCODER);
         launcher.setMode(DcMotor.RunMode.STOP_AND_RESET_ENCODER);
         launcher.setMode(DcMotor.RunMode.RUN_USING_ENCODER);
-        addTask(new DeadmanMotorTask(this, launcher, 0.1, GamepadTask.GamepadNumber.GAMEPAD_1, DeadmanMotorTask.DeadmanButton.BUTTON_Y));
-        addTask(new DeadmanMotorTask(this, launcher, -0.1, GamepadTask.GamepadNumber.GAMEPAD_1, DeadmanMotorTask.DeadmanButton.BUTTON_A));
 
-        stt = new SingleShotTimerTask(this, 1000);
+        /*
+        runLauncherForwardTask = new DeadmanMotorTask(this, launcher, 0.1, GamepadTask.GamepadNumber.GAMEPAD_1, DeadmanMotorTask.DeadmanButton.BUTTON_Y);
+        runLauncherBackTask = new DeadmanMotorTask(this, launcher, -0.1, GamepadTask.GamepadNumber.GAMEPAD_1, DeadmanMotorTask.DeadmanButton.BUTTON_A);
+        runConveyorForwardTask = new DeadmanMotorTask(this, conveyor, 0.1, GamepadTask.GamepadNumber.GAMEPAD_1, DeadmanMotorTask.DeadmanButton.BUTTON_X);
+        runConveyorBackTask = new DeadmanMotorTask(this, conveyor, -0.1, GamepadTask.GamepadNumber.GAMEPAD_1, DeadmanMotorTask.DeadmanButton.BUTTON_B);
+
+        addTask(runLauncherForwardTask);
+        addTask(runLauncherBackTask);
+        addTask(runConveyorForwardTask);
+        addTask(runConveyorBackTask);
+        */
+
+        stt = new SingleShotTimerTask(this, 2000);
+        launched = false;
     }
 
     @Override
     public void start()
     {
-        addTask(deadReckonTask);
+        /*
+        runLauncherForwardTask = new DeadmanMotorTask(this, launcher, 0.0, GamepadTask.GamepadNumber.GAMEPAD_1, DeadmanMotorTask.DeadmanButton.BUTTON_Y);
+        runLauncherBackTask = new DeadmanMotorTask(this, launcher, -0.0, GamepadTask.GamepadNumber.GAMEPAD_1, DeadmanMotorTask.DeadmanButton.BUTTON_A);
+        runConveyorForwardTask = new DeadmanMotorTask(this, conveyor, 0.0, GamepadTask.GamepadNumber.GAMEPAD_1, DeadmanMotorTask.DeadmanButton.BUTTON_X);
+        runConveyorBackTask = new DeadmanMotorTask(this, conveyor, -0.0, GamepadTask.GamepadNumber.GAMEPAD_1, DeadmanMotorTask.DeadmanButton.BUTTON_B);
+
+        addTask(runLauncherForwardTask);
+        addTask(runLauncherBackTask);
+        addTask(runConveyorForwardTask);
+        addTask(runConveyorBackTask);
+        */
```

Comment:
This is clever, but as I mentioned to Elizabeth this evening if you touch a controller during autonomous the ref will assume you are cheating.  So all positioning and match setup needs to be in an independent opmode that you run while you are queuing.  That's why this is commented out for league meet 0.